client/src/components/Sidebar/SidebarPanel.scss

```diff
@@ -13,7 +13,6 @@
   z-index: 400;
   display: flex;
   flex-direction: column;
-  overflow: hidden;
```

Comment:
Removing this allows menu items to have more than one level of nesting. I think this `overflow: hidden` was there for either one of these reasons:

- We wanted to make sure that long menu item labels are truncated.
  If that's the case, well apparently this line isn't what makes it work. Truncation of long menu items works fine with this line removed, because the `overflow: hidden` need to be applied on each menu item instead of the overall sidebar container, which is already done.
- We explicitly want to **disallow** having more than one level of nesting in the menu.
  I don't see why we need to do this? I can see us not recommending or saying that this is officially supported, but to remove the ability to do so when it could work just fine seems unnecessary to me. Especially with the fact that we don't put such restrictions on the Python side of the code.